server/src/tests/integration.rs

```diff
@@ -275,6 +276,7 @@ fn init_and_start_op_geth() -> Result<Child> {
         .output()
         .context("Call to state dump failed")?;
     check_output(output);
+    let op_geth_logs = std::fs::File::create("op_geth.log").unwrap();
```

Comment:
nit: We can just say `File::create` instead of `std::fs::File::create` since `File` is imported.